src/pybind/mgr/requirements.txt

```diff
@@ -8,4 +8,5 @@ pyOpenSSL
 execnet
 remoto
 Jinja2
+cherrypy
```

Comment:
Same here, we need to add python3-cherrypy to the ceph-mgr-cephadm package dependency list.